wiki/People/The_Team/Developers/en.md

```diff
@@ -51,11 +48,62 @@ The osu! development effort is lead by ![][flag_AU] **[peppy](https://osu.ppy.sh
 
 - ![][flag___] [osu!team](https://osu.ppy.sh/users/4341397)
 
+### Contributors
+
+Currently, the main osu! projects are hosted on GitHub and available for anyone to contribute. These are the users who have contributed significantly to these projects and played a vast role in their development.
+
+#### osu!
+
+*For detailed insights on contributions to the osu! project, refer to [this page](https://github.com/ppy/osu/graphs/contributors "GitHub").*
+
+- ![][flag_BY] [EVAST](https://osu.ppy.sh/users/8195163)
+- ![][flag_CA] [DrabWeb](https://osu.ppy.sh/users/6946022)
+- ![][flag_SA] [frenzibyte](https://osu.ppy.sh/users/14210502)
+- ![][flag_CN] [huoyaoyuan](https://osu.ppy.sh/users/2428732)
+- ![][flag_PL] [spaceman\_atlas](https://osu.ppy.sh/users/3035836)
+
+#### osu! framework
+
+*For detailed insights on contributions to the osu! framework project, refer to [this page](https://github.com/ppy/osu-framework/graphs/contributors "GitHub").*
```

Comment:
https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#for-see
```suggestion
*For detailed insights on contributions to the osu!framework project, see: [this page](https://github.com/ppy/osu-framework/graphs/contributors "GitHub")*
```
Not sure about "this page", I thought that link text should be avoided.